src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,37 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({
+  title,
+  body,
+  user,
+  comments,
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {body}
+    </p>
+
+    {comments.length > 0 ? (
+      <CommentList comments={comments} />
+    ) : (
+      <>
+        <hr />
+        <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
Avoid using 'b' HTML tag as it's considered deprecated in HTML5. Consider using 'strong' instead for better semantics and accessibility or just add styles